website/docs/installing-podman/installing-podman-on-windows.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 1
+title: Installing Podman on Windows
+description: Podman Desktop can assist you to install Podman on Windows.
+tags: [podman-desktop, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows]
+---
+
+# Installing Podman on Windows
+
+On Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+To install Podman you can:
+
+* [Install Podman in Podman Desktop](#installing-podman-in-podman-desktop)
+* [Install the Podman Preset in OpenShift Local](#installing-the-podman-preset-in-openshift-local)
+
+
+## Installing Podman in Podman Desktop
+
+Podman Desktop can assist you to install the Podman container engine on the Windows Subsystem for Linux version 2 (WSL 2).
+Main benefits are:
+
+* Ease of use
+* Windows native virtualization performance
+
+#### Prerequisites
+
+* 6 GB RAM
+* No VPN
+* User with administrator privileges
+* Windows 10 Build 18362 or greater, or Windows 11
+* Windows 64bit
+* [Hyper-V enabled](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v)
+* WSL 2 feature enabled, but no Linux virtual machine created. See [Installing WSL 2, step 1 to 5](https://docs.microsoft.com/en-us/windows/wsl/install-manual)
```

Comment:
@slemeur is that correct?

Reply:
I don't know if that's a requirement. @gbraad wdyt?